README.md

```diff
@@ -1,54 +1,279 @@
+# or ssh: git clone git@github.com:TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+msbuild /p:Configuration=Release BizHawk.sln
+```
+
+Remove the `/p:...` flag from MSBuild if you want debugging symbols.
+
+If your distro isn't listed under *Installing* above, `libblip_buf` probably isn't in your package repos. You can easily [build it yourself](https://gitlab.com/TASVideos/libblip_buf/blob/unified/README.md).
+
+Once built, see the *Installing* section, substituting the repo's `output` folder for the download.
+
+Again, if your distro isn't listed there, you might get an "Unknown distro" warning in the terminal, and BizHawk may not open or may show the missing dependencies dialog. You may need to add your distro to the case statement in the script, setting `libpath` to the location of `d3dx9_43.dll.so` (please do share if you get it working).
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+## Usage
+
+TODO
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+### TASing
+
+This section refers to BizHawk specifically. For resources on TASing in general, see [Welcome to TASVideos](http://tasvideos.org/WelcomeToTASVideos.html).
+
+TODO
+
+[Commandline](http://tasvideos.org/Bizhawk/CommandLine.html)
+
+[CompactDiscInfoDump](http://tasvideos.org/Bizhawk/CompactDiscInfoDump.html)
+
+[Rerecording](http://tasvideos.org/Bizhawk/Rerecording.html)
+
+[TAS movie file format](http://tasvideos.org/Bizhawk/TASFormat.html)
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+### Cores
+
+A *core* is what we call the smaller bits of software that emulate just one system or family of systems, e.g. NES/Famicom. For the most part, there's a "best" core for each system, based on accuracy, but there are a few alternative cores which are *faster and less accurate*.
+
+System | Core | Alt. Core | Special Core
+--:|:--|:--|:--
+Apple II | Virtu ||
+Atari 2600 | Atari2600Hawk ||
+Atari 7800 | A7800Hawk ||
+Atari Lynx | Handy ||
+Commodore 64 | C64Hawk ||
+ColecoVision | ColecoHawk ||
+Game Boy / Color | GBHawk | Gambatte | SameBoy (on SNES with SGB)
+Game Boy Advance | mGBA | VBA-Next |
+IntelliVision | IntelliHawk ||
+N64 | Mupen64Plus ||
+Neo Geo Pocket / Color | NeoPop || Dual NeoPop (dual instance)
+NES | NesHawk | QuickNes ||
+PC-FX | T.S.T. ||
+Playstation (PSX) | Octoshock ||
+Sega Game Gear | SMSHawk ||
+Sega Genesis | Genplus-gx ||
+Sega Master System | SMSHawk ||
+Sega Saturn | Saturnus ||
+Sega Pico | PicoDrive ||
+SNES | BSNES | Snes9x |
+TI-83 | TI83Hawk ||
+TurboGrafx / SuperGrafx | PCEHawk ||
+Virtual Boy | Virtual Boyee ||
+WonderSwan / Color | Cygne ||
+ZX Spectrum | ZXHawk ||
+
+Amstrad CPC, Magnavox Odyssey², PSP, and Uzebox emulation is a work-in-progress and there is **no ETA**. Cores for other systems are only conceptual. If you want to help speed up development, ask on IRC (see *Support* below).
```

Comment:
Uzebox must be released, we have published movies for it at tasvideos.

Reply:
Well it's still missing from the Open Rom filters, fixed. We're up to 26 systems from 22 somehow